src/mumble/AudioWizard.cpp

```diff
@@ -19,10 +19,11 @@
 
 #include <cmath>
 
-AudioWizard::AudioWizard(QWidget *p) : QWizard(p) {
+AudioWizard::AudioWizard(QWidget *p) : QWizard(p), m_chord(AudioOutputToken::None()) {
```

Comment:
```suggestion
AudioWizard::AudioWizard(QWidget *p) : QWizard(p), m_chord() {
```
Use new default-constructor